mlflow/mleap.py

```diff
@@ -2,6 +2,10 @@
 The ``mlflow.mleap`` module provides an API for saving Spark MLLib models using the
 `MLeap <https://github.com/combust/mleap>`_ persistence mechanism.
```

Comment:
Btw, The new added spark ml connect component also achieves the goal that run model inference locally without spark context, but note that it does not support legacy spark ML models.

Reply:
I think we should definitely add in the support once the Spark3.5 changes are ready for default-on (hopefully this quarter :) )